lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb

```diff
@@ -269,7 +269,7 @@ def validate_inclusion_of(attr)
       # @private
       class ValidateInclusionOfMatcher < ValidationMatcher
         BLANK_VALUES = ['', ' ', "\n", "\r", "\t", "\f"].freeze
-        ARBITRARY_OUTSIDE_STRING = 'shoulda-matchers test string'.freeze
+        ARBITRARY_OUTSIDE_STRING = Shoulda::Matchers.name
```

Comment:
I am not sure whether we should use `Shoulda::Matchers.name` here. I feel like this would be confusing in the message that appears when the matcher fails. Also, I am not sure that we should use this for _all_ uses of the matcher.

What do we feel about making a new variable, `ARBITRARY_OUTSIDE_CLASS_NAME`, creating a new class that lives inside of the Shoulda::Matchers namespace (maybe call it `ExampleClass`?) and then using the name of that class for the value?

Reply:
I think it's a great idea. I will work on it this weekend. I think I will have some free time to release the next version. Thanks for the review!

I tried to add some logic where the matcher would only use a class if the column was being used by a polymorphic association, but I ran into some issues. I decided to follow this path because it was much simpler. I can take another look later and try again. Maybe in another PR if you don't mind.

Reply:
Oh alright. No worries then, that sounds good.

Reply:
Thank you!